lib/elixir/src/elixir_aliases.erl

```diff
@@ -81,7 +81,10 @@ ensure_loaded(Meta, Ref, E) ->
   catch
     error:undef ->
       Kind = case lists:member(Ref, ?m(E, context_modules)) of
-        true  -> scheduled_module;
+        true  -> case ?m(E, module) of
```

Comment:
Would be better to put `case` on the next line, as it's the style throughout the repo.